src/runtime_src/core/include/ert.h

```diff
@@ -625,6 +625,12 @@ uint32_t ert_base_addr = 0;
 # define ERT_INTC_CU_96_127_ADDR           0x3000
 #endif
 
+/**
+ * This defines ERT packet version which need to sync between Host and ERT
+ */
+#define ERT_CUSTAT_VERSION_0 		  0x51a10000
+#define ERT_CUSTAT_VERSION_1		  0x51a10001
```

Comment:
Thank you for added these.  It does make the code much easier to read.

Reply:
Thanks.